components/webext-storage/src/db.rs

```diff
@@ -227,6 +268,10 @@ pub mod test {
         let counter = ATOMIC_COUNTER.fetch_add(1, Ordering::Relaxed);
         StorageDb::new_memory(&format!("test-api-{}", counter)).expect("should get an API")
     }
+
+    pub fn new_shared_mem_db() -> Arc<ThreadSafeStorageDb> {
```

Comment:
nitpick: `new_thread_safe_storage_mem_db()`? `shared_db` is shorter, but feels a little out of place with `ThreadSafeStorageDb`.